### ISSUE TITLE
Support for `/path/<url>/<login>` format

### ISSUE THREAD

**Issue by czardien** [passff/passff#455]
### General information. Describe your environment

- Versions
  - Operating system: Archlinux, Linux 5.8.12
  - Browser: Firefox 81.0
  - PassFF: 1.10.3
  - Host app: 1.2.1-3

Status line output: `[hh:mm:ss] show -> (0)`

`[17:14:50] show -> (0) no error message`

- Output of `pass`: 

```
Password Store
└── www
    └── miro.com
        └── adrien.czerny@example.com
```

### Actual behaviour

From the toolbar menu I navigate to my entry:

`www/miro.com/firstname.surname@example.com` 

When clicking Goto I am redirected to `example.com` instead of `miro.com`

### Expected behaviour

I should be redirected to `miro.com` instead of `example.com`.

### Steps to reproduce the behaviour
1. Use a login that contains special character: `@` 
2. From toolbar menu navigate to entry
3. Select Goto, watch yourself get redirected to your login's email's host instead of the URL in the path: `/path/<url>/<login>`

**Comment by czardien** [comment]
To be honest that seems like such a common use-case that I'm pretty sure I am missing something.

However I cannot find any references to that use-case anywhere in the docs and in the issues to be honest which is why I'm raising.

**Comment by tuxor1337** [comment]
What is the content of that pass entry? The directory name is only taken as URL if the file contains no URL information.

**Comment by czardien** [comment]
Hey @tuxor1337 thanks for the reply! 

The content of the pass entry is the password of the site, that is mire.com for login firstname.surname@example.com.

**Comment by czardien** [comment]
From the README this was my understanding: 

```
Example (More than one entry per hostname): If you happen to have several credentials for one hostname, you could name a directory in your password store after the hostname and list the different credentials inside that directory. Suppose you have different credentials for each of the following URLs:
https://my.example.com/cloud https://my.example.com/blog?login https://my.example.com/blog?admin 
You could store them in your password store as
/some/path/my.example.com/cloud /some/path/my.example.com/blog-login /some/path/my.example.com/blog-admin 
However, the following will work equally well:
/business/cloud/my.example.com /personal/my.example.com-blog-login /personal/my.example.com-blog-admin
```

**Comment by czardien** [comment]
Also worth noting that passff successfully autocompletes my login form when I go to that website from that entry.

**Comment by tuxor1337** [comment]
This is from the README section about improving the suggestions. Currently, an item's URL is only read from the corresponding url field. I convert this issue into a feature request.

**Comment by czardien** [comment]
Ha! Thanks for the clarification.

I thought the docs were quite confusing on that point. I just tried multiple solutions until I found this to be an acceptable solution, which the **Goto** and **Copy** features would handle well:

```
$ pass -m insert www/miro.com/email@example.com

login: email@example.com
password: ********
url: https://miro.com/login
```

Although if I have to be honest I feel like I am not sure it is behaving well, since I've had my browser filling login forms on other sites (github for instance) even though they are not in my pass store. I am considering raising another issue for that.
